### DISCUSSION TITLE
Get note from midi based on time. (UNITY)

### DISCUSSION THREAD

**Discussion by KartMakerBrosU** [melanchall/drywetmidi#304] [Q&A]
Say I have a button and a timer. Whenever i click the button, I want to be able to get the timer time, and find its corresponding note (in MIDI note) that would be in the MIDI, or the last played note before that time. How would I do this?

**Comment by melanchall** [comment]
> I want to be able to get the timer time

What is "timer"? There is no such thing in the DryWetMIDI library.

> find its corresponding note (in MIDI note)

Find note in note?

Honestly, I didn't understand anything. What do you want to do?

Please read [this article](https://melanchall.github.io/drywetmidi/articles/dev/Support.html) and correct your question.

**Reply by KartMakerBrosU** [reply]
Sorry, I wanted to find the note of the midi( one channel) based on a time variable (in seconds) then retrieve its midi note number. You don't have to reply to this as I've already figured out a solution to this. So all good 👍
